curriculum/challenges/english/25-front-end-development/lab-real-time-counter/66bb3e20d3dc5b6d0a21f5dd.md

```diff
@@ -65,7 +65,8 @@ const charCount = document.getElementById('char-count');
 textInput.value = "I am learning a new language and it is called c++.";
 textInput.dispatchEvent(new Event('input'));
 textInput.dispatchEvent(new Event('change'));
-assert.strictEqual(charCount.style.color, 'red');
+const computedStyle = window.getComputedStyle(charCount);
+assert.strictEqual(computedStyle.color, 'red');
```

Comment:
See the failed test, you can't use `red` here. Maybe you can use `assert.oneOf` and all the possible ways to have the `red` colors to avoid differences between browser implementations